### ISSUE TITLE
Having some issues with php/sql related to softaculous and myvesta

### ISSUE THREAD

**Issue by shillshocked** [myvesta/vesta#47]
Trying to reinstall my Wordpress site on a new server remotely through Softaculous. Error that it can't connect to database. I think it might be because "disabled dangerous PHP functions in php.ini." Any way to revert the changes temporarily to do the import?

**Comment by akz1** [comment]
If i'm not wrong vesta uses own php (5.6) vesta-php package (+ioncube) in admin panel which is used to run softaculous. 
`/usr/local/vesta/php/lib/php.ini` and there is not disable_functions= in.

php.ini for sites is located at /etc/php/7.X/fpm/php.ini

I'm able to install wordpress on myvesta fresh install with multiple php and with more disabled functions without any error, could you explain how i can reproduce your issue?

**Comment by shillshocked** [comment]
I'm trying to import from another site. Also, I downloaded it and tried a manual install. Neither succeeded. Furthermore, with the file manager plugin, I can't even see the files I copied, although they exist if I navigate the directories via ssh.

**Comment by dpeca** [comment]
When did you installed Softaculous?

**Comment by shillshocked** [comment]
I just installed yesterday. I was able to make some progress by changing the permissions of some of my files and folders. However, I'm stuck with this error on manual import:

`Could not connect to database, please check the Username/Password of your database. If changed, please update the details in Softaculous installations using Edit Details.`

I've reset my mysql password and updated it in the manual directories, as well as check that I have the correct password, and yet I still get this error.

**Comment by shillshocked** [comment]
Alright, I managed to fix it by updating with the new database credentials in wp-config.php

**Comment by dpeca** [comment]
And fresh installation of WordPress works fine?

**Comment by shillshocked** [comment]
Yes, wordpress functions normally. The problem is importing with softaculous remotely.

**Comment by dpeca** [comment]
@shillshocked 
Hm, not sure I understand you.

I asked - if you go to Softaculous and choose WP and press "Install" on an empty domain, does it works?

What is ''remotely'' import?
Where is it in Softaculous?

In other words, describe me exactly what you are doing, step-by-step, so I can reproduce error.

**Comment by shillshocked** [comment]
You have to go into Apps - Wordpress - Import - Remote Import
